apps/ui/src/components/FormNetwork.vue

```diff
@@ -1,13 +1,13 @@
 <script setup lang="ts">
-import { enabledNetworks, getNetwork } from '@/networks';
+import { enabledSpaceNetworks, getNetwork } from '@/networks';
 import { getUrl } from '@/helpers/utils';
 import type { NetworkID } from '@/types';
 
 const model = defineModel<NetworkID>({
   required: true
 });
 
-const availableNetworks = enabledNetworks
+const availableNetworks = enabledSpaceNetworks
```

Comment:
The idea was to have single list of for all usages, now instead of reusing singe list we additionally filter it for second time (with the same effect as of now).

1. I think we should filter it by `readOnly` flag, same as it was in `FormNetwork`. We should filter by capabilities, not by some hardcoded arrays.
2. `enabledSpaceNetworks` is confusing to me, this name doesn't tell me how to interpret it at all. If we follow 1. then `enabledReadWriteNetworks` is clearer.

Reply:
I tried the `enabledReadWriteNetworks` route, using the same logic from `FormNetwork`.

The issue now is that `enabledReadWriteNetworks` is now a `Record<NetworkID, Network>`, which may be confusing with `enabledNetworks`, which is an array of `NetworkID`. Maybe should find a better naming for that.

This new object is heavier, but remove the usage for `getNetwork()` in `Create.vue` and `FormNetwork.vue`

Reply:
> The issue now is that enabledReadWriteNetworks is now a Record<NetworkID, Network>, which may be confusing with enabledNetworks

Why is it Record and not an array, I was thinking array made sense. I don't think that `getNetwork` usage is bad either.

Reply:
Fixed, don't know why I had this idea